### PR TITLE
Added Clipperton Island to ISO standard

### PR DESCRIPTION
**Pull request by bdeining**

### CODE REVIEW

**Comment by bdeining** [comment]
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#CP

**Review by peterhuffer — APPROVED** [review]

**Comment by peterhuffer** [comment]
image

File: standards/iso/src/test/groovy/org/codice/countrycode/standards/iso/Iso3166StandardProviderSpec.groovy [hunk 1]
```diff
@@ -19,6 +19,6 @@ class Iso3166StandardProviderSpec extends Specification {
 
     def 'test iso3166 provider'() {
         expect:
-        new Iso3166StandardProvider().getStandardEntries().size() == 257
```
Comment (rymach) [inline standards/iso/src/test/groovy/org/codice/countrycode/standards/iso/Iso3166StandardProviderSpec.groovy:22]: we should delete this test

**Review by rymach — APPROVED** [review]

**Review by rymach — APPROVED** [review]

**Comment by rymach** [comment]
I don't remember how it works, but you may want to take a look at these files 
https://github.com/codice/countrycode/tree/master/standards/iso/src/main/resources

**Comment by bdeining** [comment]
@rymach i added it to https://github.com/codice/countrycode/blob/3d218f030763c60f2d2f6061fcbbe32cd5b63eb7/standards/iso/src/main/resources/iso3166-1.json

File: standards/iso/src/test/groovy/org/codice/countrycode/standards/iso/Iso3166StandardProviderSpec.groovy (hunk 1 above)
Comment (bdeining) [reply standards/iso/src/test/groovy/org/codice/countrycode/standards/iso/Iso3166StandardProviderSpec.groovy:22]: i'm not sure about that; this forces us to think about augmenting the json

**Review by Bdthomson — APPROVED** [review]

**Pull request merged**